packages/expo-updates/ios/EXUpdates/AppLoader/EXUpdatesAsset.m

```diff
@@ -22,7 +22,7 @@ - (nullable NSString *)filename
 {
   if (!_filename) {
     if (_key) {
-      _filename = _key;
+      _filename = [NSString stringWithFormat:@"%@.%@", _key, _type];
```

Comment:
Still need to make the analogous change on Android here (unless we intentionally want to store files extension-less on Android for some reason). I think this lives in `UpdatesUtils` on Android.

Reply:
Lol, how did I miss this. 😅